source/components/handling-events.md

```diff
@@ -0,0 +1,129 @@
+Instead of having to register event listeners on elements you'd like to
+respond to, simply implement the name of the event you want to respond to
+as a method on your component.
+
+For example, imagine we have a template like this:
+
+```hbs
+{{#x-clickable}}
+  This is a clickable area!
+{{/x-clickable}}
+```
+
+Let's implement `x-clickable` such that when it is
+clicked, an alert is displayed:
+
+```app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    alert("XClickableComponent was clicked!");
+  }
+});
+```
```

Comment:
Since clicking is the most common action and can be handled in other ways, I'd use doubleClick or some other event to make the use case for this feature more obvious.